src/payment/payment.ts

```diff
@@ -80,6 +82,10 @@ export interface PaypalInstrument {
     };
 }
 
+export interface FormattedBrowserInfo {
```

Comment:
Is the idea that we can send just `FormattedBrowserInfo`? I would expect this is in addition to the rest of the payment information.

Reply:
Yes, as very few providers require this info to be sent, I believe it would be better to be able to send it only when necessary. Also if the we send the browser info outside the formattedPayload it would require the mappers to be changed as well to allow the browser info fields to be sent.